src/dataLoaders/components/collectorsWizard/select/StreamingSelector2.tsx

```diff
@@ -158,9 +158,16 @@ class StreamingSelectorTelegrafUiRefresh extends PureComponent<Props, State> {
   private get filteredBundles(): TelegrafPlugin[] {
     const {searchTerm} = this.state
 
-    return WRITE_DATA_TELEGRAF_PLUGINS.filter(b =>
-      b.name.toLowerCase().includes(searchTerm.toLowerCase())
-    )
+    return WRITE_DATA_TELEGRAF_PLUGINS.sort((plugin1, plugin2) => {
+      // sort the plugins array alphabetically
+      if (plugin1.name.toLowerCase() < plugin2.name.toLowerCase()) {
+        return -1
+      }
+      if (plugin1.name.toLowerCase() > plugin2.name.toLowerCase()) {
+        return 1
+      }
+      return 0
+    }).filter(b => b.name.toLowerCase().includes(searchTerm.toLowerCase()))
```

Comment:
for consistency, it'll probably be easier to follow if this `b` was renamed to `plugin` since that seems to be what the object being passed to the `filter` callback is, and the `sort` function uses `plugin`.

Reply:
I agree! I'll rename it!